eng/common/scripts/Create-APIReview.ps1

```diff
@@ -153,6 +153,13 @@ function Submit-APIReview($packageInfo, $packagePath)
     }
 }
 
+function IsApiviewStatusCheckRequired($packageInfo)
```

Comment:
Non-blocking question, is this something that should be moved into the `Language-Settings.ps1` pattern? That way the language repo can update handling without requiring azure-sdk-tools to be changed. Or, is it done this way to prevent that exact scenario?

Reply:
That's the plan to introduce this function as a wrapper and we will add a supporting function in language settings to drive which type of SDK  require api status check.